securedrop-admin

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python2
+"""
+SecureDrop Admin Toolkit.
+
+For use by administrators to install, maintain, and manage their SD
+instances.
+"""
+
+import argparse
+import subprocess
+import os
+import sys
+import getpass
+import logging
+
+sdlog = logging.getLogger('SD')
+
+SD_DIR = os.path.dirname(os.path.realpath(__file__))
+ANSIBLE_PATH = os.path.join(SD_DIR, "./install_files/ansible-base")
+SITE_CONFIG = os.path.join(ANSIBLE_PATH, "group_vars/all/site-specific")
+VENV_DIR = os.path.join(SD_DIR, ".venv")
+VENV_ACTIVATION = os.path.join(VENV_DIR, 'bin/activate_this.py')
+
+
+def logger_setup(verbose=False):
+    """ Configure logging handler """
+    # Set default level on parent
+    sdlog.setLevel(logging.DEBUG)
+    level = logging.DEBUG if verbose else logging.INFO
+
+    stdout = logging.StreamHandler(sys.stdout)
+    stdout.setFormatter(logging.Formatter('%(levelname)s: %(message)s'))
+    stdout.setLevel(level)
+    sdlog.addHandler(stdout)
+
+def activate_venv():
+    """Use to activate the local virtualenv"""
+    try:
+        execfile(VENV_ACTIVATION, dict(__file__=VENV_ACTIVATION))
+    except IOError:
+        sdlog.error("Pre-requisites not in place, re-run command with 'setup'")
+        sys.exit(1)
+
+
+def sdconfig(args):
+    """Configure SD site settings"""
+    activate_venv()
+    subprocess.check_call([os.path.join(ANSIBLE_PATH,'configure')],
+                            cwd=ANSIBLE_PATH, env=os.environ.copy())
+
+
+def envsetup(args):
+    """Install SD pre-reqs"""
+    # virtualenv doesnt exist? Install dependencies and create
+    if not os.path.exists(VENV_ACTIVATION):
+        sdlog.info("Installing dependencies. You'll be prompted for the "+\
```

Comment:
Nit: You don't need the `+\` at the end here. The resulting string will be the same either way, but it's more PEP8-compliant to remove those characters.